skyportal/handlers/api/gcn.py

```diff
@@ -221,24 +221,44 @@ def delete(self, dateobs):
         return self.success()
 
 
-def add_contour(localization_id, request_handler):
+def add_tiles(localization_id):
+    session = Session()
     try:
-        localization = (
-            Localization.query_records_accessible_by(request_handler.current_user)
-            .filter(
-                Localization.id == localization_id,
+        localization = session.query(Localization).get(localization_id)
```

Comment:
Permission checking removed; is this safe?

Reply:
This is just adding the Localization tiles, so you would only be doing this on your own localization. Is there an equivalent session query with the accessible bit?